cluster-autoscaler/cloudprovider/azure/azure_manager.go

```diff
@@ -78,9 +93,25 @@ type AzureManager struct {
 	explicitlyConfigured  map[string]bool
 }
 
+// CloudProviderRateLimitConfig indicates the rate limit config for each clients.
+type CloudProviderRateLimitConfig struct {
+	// The default rate limit config options.
+	azclients.RateLimitConfig
+
+	// Rate limit config for each clients. Values would override default settings above.
+	InterfaceRateLimit              *azclients.RateLimitConfig `json:"interfaceRateLimit,omitempty" yaml:"interfaceRateLimit,omitempty"`
+	VirtualMachineRateLimit         *azclients.RateLimitConfig `json:"virtualMachineRateLimit,omitempty" yaml:"virtualMachineRateLimit,omitempty"`
+	StorageAccountRateLimit         *azclients.RateLimitConfig `json:"storageAccountRateLimit,omitempty" yaml:"storageAccountRateLimit,omitempty"`
+	DiskRateLimit                   *azclients.RateLimitConfig `json:"diskRateLimit,omitempty" yaml:"diskRateLimit,omitempty"`
```

Comment:
is this needed? Similar question for StorageAccount and VirtualMachines?

Reply:
I deleted the unneeded and keep vmss and vm. I think vm is needed, I'll update vm client in another pr.

Reply:
let's only keep rate limit configs for the clients existing in azure_client.go